rtsp_to_hls/lib/pipeline.ex

```diff
@@ -115,8 +127,20 @@ defmodule Membrane.Demo.RtspToHls.Pipeline do
 
   @impl true
   def handle_child_notification(notification, element, _ctx, state) do
-    Logger.warn("Unknown notification: #{inspect(notification)}, el: #{inspect(element)}")
+    Logger.warning("Unknown notification: #{inspect(notification)}, el: #{inspect(element)}")
 
     {[], state}
   end
+
+  @impl true
+  def handle_element_end_of_stream(Pad.ref(:input), _pad, _context, state) do
```

Comment:
This won't work, see https://hexdocs.pm/membrane_core/Membrane.Pipeline.html#c:handle_element_end_of_stream/4

Reply:
As discusses, I have removed the callback implementation